FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -18,7 +18,63 @@ public final class RemoteFeedLoader: FeedLoader {
 		self.client = client
 	}
 
+	private var OK_200: Int { return 200 }
+
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+			switch result {
+			case let .success((data, httpURLResponse)):
+
+				if httpURLResponse.statusCode != self.OK_200 {
+					completion(.failure(Error.invalidData))
+					return
+				}
+
+				do {
+					let value = try JSONDecoder().decode(RemoteFeedItem.self, from: data)
+					var feedImageList: [FeedImage] = []
+					value.items.forEach { item in
+						feedImageList.append(FeedImage(id: item.id, description: item.description, location: item.location, url: item.url))
+					}
+					completion(.success(feedImageList))
+
+				} catch {
+					completion(.failure(Error.invalidData))
+					return
+				}
+
+			case .failure(_):
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+
+	private class RemoteFeedItem: Decodable {
+		let items: [RemoteFeedImage]
+		init(items: [RemoteFeedImage]) {
+			self.items = items
+		}
+	}
+
+	private class RemoteFeedImage: Decodable {
```

Comment:
See my previous comment:
```suggestion
	private struct RemoteFeedImage: Decodable {
```